site/en/docs/extensions/mv3/favicon/index.md

```diff
@@ -1,10 +1,10 @@
 ---
 layout: "layouts/doc-post.njk"
 title: "Fetching favicons"
-seoTitle: "Fetching with favicons in Chrome extensions"
+seoTitle: "Fetching favicons in Chrome extensions"
 date: 2023-01-11
 # updated: 2023-02-14
-description: How to get a website's favicon in a Chrome extension.
+description: How to get a website's favicon.
```

Comment:
I like this change. Information that's implicit by context has limited utility, I think.